lore/estimators/naive.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+"""
+Naive Estimator
+****************
+A naive estimator is a useful baseline against which to benchmark more complex models.
+A naive estimator will return the mean of the outcome for regression models and
+the majority class for classification models.
```

Comment:
majority or plurality?

Reply:
Plurality for multi-class. But I've only implement a binary classifier for now